docs/book/features/volume_health.md

```diff
@@ -0,0 +1,81 @@
+# vSphere CSI Driver - Volume Health
+Volume Health feature provides the health status of persistent volume. Currently, Volume Health is only supported for vSphere with Kubernetes Cluster and Tanzu Kubernetes Grid Cluster and only supported for block volume.
+
+## Health Status of Persistent Volume
+Currently, Volume Health only report the health status for Persistent Volumes which have been provisioned successfully. For each Persistent Volume in `bound` state, the health status of this Persistent Volume will be exposed as an annotation of Persistent Volume Claim which is bound to the Persistent Volume. The key of the annotation is `volumehealth.storage.kubernetes.io/messages:`. There are two possible values for the health status.
+
+1. `accessible`:  The Persistent Volume is accessible and avaialbe for use.
+2. `inaccessible`: The Persistent Volume is inaccessible, user should avoid using this Persistent Volume. Persitent Volume goes to `inaccessible` if the datastore that hosts the Persistent Volume is not accessible from all hosts it is mounted.
+
+## Volume Health Status Example
+
+Create a PVC in the Tanzu Kubernetes Grid Cluster.
+
+```bash
+apiVersion: v1
```

Comment:
It would be better to add - `cat example-pvc.yaml` before here. as in the following command, you are using the filename to create PVC.